src/lms7002m/LMS7002M.cpp

```diff
@@ -298,10 +281,9 @@ size_t LMS7002M::GetActiveChannelIndex(bool fromChip)
 
 OpStatus LMS7002M::EnableChannel(TRXDir dir, const uint8_t channel, const bool enable)
 {
-    ChannelScope scope(this, channel);
+    ChannelScope scope(this, channel, false);
 
-    const Channel ch = this->GetActiveChannel();
-    this->SetActiveChannel(channel > 0 ? Channel::ChB : Channel::ChA);
+    const Channel ch = channel > 0 ? Channel::ChB : Channel::ChA;
```

Comment:
this conversion from int to channel happens a lot, might as well make a function for that.